creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.ts

```diff
@@ -623,18 +614,13 @@ const _additionalSyncIsRequired = async (
     }
   }
 
-  if (syncStatus?.startsWith('success')) {
-    await SecondarySyncHealthTracker.recordSuccess(
+  if (!syncStatus.startsWith('success')) {
```

Comment:
this wont ever be null right? i saw that there was a `?` in the before code

Reply:
if there is no sync status, should we record this as a fail? or did a sync just not happen

Reply:
the sync happened if we reach this point so I'd say record as a fail. maybe the reason can be its own reason like "node_returned_null_status"